app/src/main/java/com/mobile/tandil/javabaseproject/mvp/model/ParkingModel.java

```diff
@@ -4,4 +4,15 @@
 
 public class ParkingModel implements ParkingContract.Model {
 
+    private int AvailableParkingSpots = 0;
```

Comment:
Use an invalid value as default, for example -1.

Reply:
done